README.md

```diff
@@ -86,9 +88,12 @@ The above gives us a way to add/update users after a successful handshake.  `$p
 3. Our routes need to be updated to the following:
 
 ```php
-Route::get('/login/azure', '\App\Http\Middleware\AppAzure@azure');
-Route::get('/login/azurecallback', '\App\Http\Middleware\AppAzure@azurecallback');
-Route::get('/logout/azure', '\App\Http\Middleware\AppAzure@azurelogout');
+Route::get('/login/azure', '\App\Http\Middleware\AppAzure@azure')
+    ->name('azure.login');
+Route::get('/login/azurecallback', '\App\Http\Middleware\AppAzure@azurecallback')
```

Comment:
I would just add a note saying `if we are using named routes, be sure to add the named routes here appropriately.`